samples/openshift/gitops/details.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 Istio Authors
```

Comment:
I feel the bookinfo app is an overkill for us to demonstrate Iter8 related concepts.

In addition to complexity of GitOps / CI / CD concepts, the app is adding needless complexity through multiple services, deployments and gateways.

The hello world application at that is used in the getting started tutorial, and also Iter8's other GitOps tutorial for that matter is a whole lot simpler. In light of the [simplicity of this workflow](https://iter8.tools/0.7/tutorials/deployments/slo-validation-gitops/), do you think the OpenShift GitOps tutorial can be further simplified? In particular, using hello world instead of bookinfo? 

Same is true for all of our Istio tutorials. While bookinfo is excellent for demonstrating Istio, I'm not sure why we need all that complexity for Iter8.

Reply:
I agree with this comment. However, I want to show some screenshots of the app itself on the blog article that I'm writing, so it's nice to have a prettier app that looks more like a real app than the helloworld app. Also, if you look at `...samples/openshift/gitops`, there are only a few yaml files, and they're all encapsulated by `argocd-app.yaml`, so it's already pretty simple.